docs/semgrep-secrets/historical-scanning.md

```diff
@@ -0,0 +1,119 @@
+---
+slug: historical-scanning
+append_help_link: true
+title: Scan your Git history (beta)
+hide_title: true
+description: Detect valid, leaked secrets in previous Git commits through a historical scan.
+tags:
+  - Semgrep Secrets
+  - Team & Enterprise tier
+---
+
+# Scan your Git history (beta)
+
+Detect valid, leaked secrets in previous Git commits through a **historical scan**.
+
+You can perform one-time historical scans or enable historical scanning for full Secrets scans. Detecting valid secrets in your Git history is a step to reducing your repository's attack surface.
```

Comment:
Would maybe say "full Semgrep scans" instead of "full Secret scans," but don't feel strongly

Reply:
Yes, how we frame the scan as either per-product vs entire-platform is a valid question. I thought about it a bit. The reason why I went with "Secrets scan" is because even though we have trifecta users, they may configure their scans on a per-product basis (`semgrep ci --secrets`).